File: auto-docs/executables/r/RdBu_heatmap.r
library(plotly)

p <- plotly(username='TestBot', key='r1neazxo9w')

size <- 50
z <- matrix(0, size, size)
for(r in 1:size) {
    for(c in 1:size) {
        z[r, c] <- sqrt(r*c/size^2)
    }
}
data <- list(
  list(
    z = z, 
    scl = "RdBu", 
    type = "heatmap"
  )
)
layout <- list(title = "RdBu")

response <- p$plotly(data, kwargs=list(layout=layout, filename="RdBu-heatmap", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/julia/RdBu_heatmap.jl
using Plotly

using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

size = 50
z = rand(size, size)
for r = 1:size
  for c = 1:size
        z(r,c) = r+c

data = [
  [
    "z" => z, 
    "scl" => "RdBu", 
    "type" => "heatmap"
  ]
]
layout = ["title" => "RdBu"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "RdBu-heatmap", "fileopt" => "overwrite", "auto_open" => "false"])
plot_url = response["url"]